log-collector-script/linux/eks-log-collector.sh

```diff
@@ -189,7 +189,12 @@ create_directories() {
 
 get_instance_metadata() {
   readonly INSTANCE_ID=$(curl --max-time 3 --silent http://169.254.169.254/latest/meta-data/instance-id 2>/dev/null)
-  echo "${INSTANCE_ID}" > "${COLLECT_DIR}"/system/instance-id.txt
+  echo "Instance ID = ${INSTANCE_ID}" >> "${COLLECT_DIR}"/system/instance-metadata.txt
+  echo "Instance Type:" $(curl --max-time 3 --silent http://169.254.169.254/latest/meta-data/instance-type 2>/dev/null) >> "${COLLECT_DIR}"/system/instance-metadata.txt
+  readonly MAC=$(curl --max-time 3 --silent http://169.254.169.254/latest/meta-data/mac/ 2>/dev/null)
+  echo "Local-IPv4s:" $(curl --max-time 3 --silent http://169.254.169.254/latest/meta-data/network/interfaces/macs/"${MAC}"/local-ipv4s 2>/dev/null) >> "${COLLECT_DIR}"/system/instance-metadata.txt
```

Comment:
This won't get all local IPv4s from the metadata, just the primary interface. Getting the metadata for all MACs is a lot more annoying. Just look at [`ec2net-functions`](https://github.com/aws/amazon-ec2-net-utils/blob/e2d4c3b74993576ddd51f10d300a19314bf8e413/ec2net-functions) in `aws/amazon-ec2-net-utils` for an example.

Maybe worth noting that in the log? `"Primary interface IPv4s:"`?